rules/sinks/third_parties/sdk/oracle/java.yaml

```diff
@@ -7,791 +7,791 @@ sinks:
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]internal).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Model
     name: Oracle Model
     domains:
-      - "model.oracle.com"
+      - "oracle.com/in/database/sqldeveloper/technologies/sql-data-modeler/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]model).*"
     tags:
     
   - id: ThirdParties.SDK.Oracle.Waiter
     name: Oracle Waiter
     domains:
-      - "waiter.oracle.com"
+      - "oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]waiter).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.ResourceSearch
     name: Oracle Resource Search
     domains:
-      - "resourcesearch.oracle.com"
+      - "oracle.com/in/technical-resources/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]resourcesearch).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.KeyManagement
     name: Oracle Key Management
     domains:
-      - "keymanagement.oracle.com"
+      - ".oracle.com/en-us/iaas/Content/KeyManagement/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]keymanagement).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.AnnouncementsService
     name: Oracle Announcements Service
     domains:
-      - "announcementsservice.oracle.com"
+      - ".oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]announcementsservice).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Healthchecks
     name: Oracle Healthchecks
     domains:
-      - "healthchecks.oracle.com"
+      - "oracle.com/cloud/networking/health-checks/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]healthchecks).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Waas
     name: Oracle Waas
     domains:
-      - "waas.oracle.com"
+      - ".oracle.com/en-us/iaas/pl-sql-sdk/doc/dbms_cloud_oci_ws_waas."
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]waas).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Monitoring
     name: Oracle Monitoring
     domains:
-      - "monitoring.oracle.com"
+      - "oracle.com/in/devops/monitoring/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]monitoring).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.ResourceManager
     name: Oracle Resource Manager
     domains:
-      - "resourcemanager.oracle.com"
+      - "oracle.com/in/devops/resource-manager"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]resourcemanager).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Ons
     name: Oracle Ons
     domains:
-      - "ons.oracle.com"
+      - "oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]ons).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Autoscaling
     name: Oracle Autoscaling
     domains:
-      - "autoscaling.oracle.com"
+      - "oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]autoscaling).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Functions
     name: Oracle Functions
     domains:
-      - "functions.oracle.com"
+      - "oracle.com/en-us/iaas/tools/java/3.4.0/com/oracle/bmc/functions"
```

Comment:
Looks like a link to a library javadoc instead of a product. Use the link to product/service instead.